test/TicketingService.IntegrationTests/JwtTokenHelper.cs

```diff
@@ -11,15 +11,18 @@ public static class JwtTokenHelper
     public static string CreateJwtToken(ClaimsIdentity claimsIdentity, TimeSpan? expirationFromUtcNow = null)
     {
         var tokenHandler = new JwtSecurityTokenHandler();
-        var key = Encoding.ASCII.GetBytes($"{nameof(JwtTokenHelper)}.{nameof(CreateJwtToken)}");
+        const string sec =
+            "401b09eab3c013d4ca54922bb802bec8fd5318192b0a75f201d8b3727429090fb337591abd3e44453b954555b7a0812e1081c39b740293f765eae731f5a65ed1";
```

Comment:
FYI: When leaving this code as is, I was getting an exception saying, `"IDX10720: Unable to create KeyedHashAlgorithm for algorithm 'http://www.w3.org/2001/04/xmldsig-more#hmac-sha256', the key size must be greater than: '256' bits, key has '232' bits. (Parameter 'keyBytes')"`